src/blocks.ts

```diff
@@ -0,0 +1,44 @@
+import axios, { AxiosResponse } from "axios";
```

Comment:
Change it to `node-fetch` for using the native Fetch API, `axios` is powerful when you are using its interceptors feature